modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -303,40 +255,36 @@ def add_error_informal_conference_rep_will_not_fit_on_form
 
     # validation (header)
     def birth_date_is_a_date
-      add_error("Veteran birth date isn't a date: #{birth_date_string.inspect}") unless birth_date
+      add_error("Veteran birth date isn't a date: #{birth_date.raw_date.inspect}") unless birth_date.valid?
```

Comment:
Curious is the `#inspect` method needs to be called here or not, because it's not in the next `add_error` call.

Reply:
Possibly not! I was just blitzing through and adding functionality parity, but I can go back and refine this